openedx/adg/lms/registration_extension/migrations/0001_initial.py

```diff
@@ -11,14 +11,17 @@ class Migration(migrations.Migration):
 
     dependencies = [
         migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('applications', '0003_user_application_update_business_line'),
     ]
 
     operations = [
         migrations.CreateModel(
             name='ExtendedUserProfile',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('company', models.CharField(blank=True, choices=[('Tanmiah', 'Tanmiah Food Group'), ('Petromin', 'Petromin'), ('RSI', 'Red Sea International'), ('Dukan', 'Dukan'), ('SAED', 'SAED'), ('BARQ', 'BARQ System'), ('PPC', 'Premium Paints Company'), ('IPD', 'International Project Developers'), ('ADG', 'Al-Dabbagh Group')], max_length=50, null=True)),
+                ('birth_date', models.DateField(verbose_name='Birth Date')),
+                ('saudi_national', models.BooleanField(verbose_name='Saudi National')),
+                ('company', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to='applications.BusinessLine', verbose_name='Company')),
                 ('user', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, related_name='extended_profile', to=settings.AUTH_USER_MODEL)),
```

Comment:
Why are we changing previous migrations? I think this migration might have already been run on dev, staging, and prod